Sources/NIO/EventLoop.swift

```diff
@@ -20,8 +20,9 @@ import Dispatch
 /// A `Scheduled` allows the user to either `cancel()` the execution of the scheduled task (if possible) or obtain a reference to the `EventLoopFuture` that
 /// will be notified once the execution is complete.
 public struct Scheduled<T> {
-    private let promise: EventLoopPromise<T>
+    /* private but usableFromInline */ @usableFromInline let promise: EventLoopPromise<T>
```

Comment:
please underscore because really it's private :)

Reply:
Done.